openstates/md/bills.py

```diff
@@ -165,10 +170,132 @@ def parse_bill_votes(self, doc, bill):
                 vote.add_source(href)
                 yield vote
 
+    def parse_bill_votes_new(self, doc, bill):
+        elems = doc.xpath('//a')
+
+        # MD has a habit of listing votes twice
+        seen_votes = set()
+
+        for elem in elems:
+            href = elem.get('href')
+            if (href and "votes" in href and href.endswith('pdf') and
+                    ("Senate" in href or "House" in href) and href not in seen_votes):
+                seen_votes.add(href)
+                vote = self.parse_vote_pdf(href, bill)
+                vote.add_source(href)
+                yield vote
+
+    def parse_vote_pdf(self, vote_url, bill):
+
+        filename, response = self.urlretrieve(vote_url)
+
+        text = convert_pdf(filename, type='text').decode()
+        lines = text.splitlines()
+
+        if 'senate' in vote_url:
+            chamber = 'upper'
+        else:
+            chamber = 'lower'
+
+        date_string = lines[0].split('Date:')[1].strip()
+        date = datetime.datetime.strptime(date_string, "%b %d, %Y %I:%M (%p)")
+
+        pageIndex = -1
```

Comment:
You can start with it at `None` instead, I think that'd be more Pythonic.

Reply:
Also, follow Pythonic variable naming, underscores not camelCase.

Reply:
Okay